docker-wrappers/LocalNeighborhood/Dockerfile

```diff
@@ -1 +1,11 @@
 # Create a Docker image for the Local Neighborhood algorithm here
+# PathLinker wrapper
+# https://github.com/Murali-group/PathLinker
+FROM python:3.10-alpine
+
+# gettext is required for the envsubst command
+# See https://github.com/haskell/cabal/issues/6126 regarding wget
+RUN apk add --no-cache ca-certificates gettext wget
```

Comment:
This is only used by PathLinker in order to add the `wget` binary so that we can download files into the container. That isn't needed for Local Neighborhood, we can remove them here.